test-apps/display-test-app/src/frontend/DisplayScale.ts

```diff
@@ -4,14 +4,14 @@
 *--------------------------------------------------------------------------------------------*/
 
 import { Matrix3d, Point3d, Transform } from "@itwin/core-geometry";
-import { IModelApp, ModelDisplayTransformProvider, Tool } from "@itwin/core-frontend";
+import { IModelApp, ModelDisplayTransform, ModelDisplayTransformProvider, Tool } from "@itwin/core-frontend";
 import { parseArgs } from "@itwin/frontend-devtools";
 
 class DisplayScaleTransformProvider implements ModelDisplayTransformProvider {
   public constructor(private readonly _models: Set<string>, private readonly _scaleTransform: Transform) { }
 
-  public getModelDisplayTransform(modelId: string): Transform | undefined {
-    return this._models.has(modelId) ? this._scaleTransform.clone() : undefined;
+  public getModelDisplayTransform(modelId: string): ModelDisplayTransform | undefined {
```

Comment:
What's all this about?

Reply:
There was a typing issue with this, but I can't seem to get the problem to recreate right now, so I'll just remove it.